torchvision/io/__init__.py

```diff
@@ -20,10 +23,75 @@
     encode_jpeg,
     write_jpeg,
     encode_png,
-    write_png
+    write_png,
 )
 
 
+if _HAS_VIDEO_OPT:
+
+    class Video:
+        """
+        Low level video-reading API.
```

Comment:
It would be good to have more details here, probably including an example on how to use it